google/cloud/bigtable/internal/common_client_test.cc

```diff
@@ -37,7 +37,9 @@ class OutstandingTimersTest : public ::testing::Test {
 };
 
 TEST_F(OutstandingTimersTest, Trivial) {
-  std::make_shared<OutstandingTimers>(std::make_shared<CompletionQueue>(cq_));
+  auto unused = std::make_shared<OutstandingTimers>(
+      std::make_shared<CompletionQueue>(cq_));
+  unused.reset();
```

Comment:
Was giving the value a name enough, or did you find you needed the explicit `reset()`?

Reply:
Perhaps a comment either way would leave a breadcrumb.

Reply:
The reset was not needed for MSVC, let's see what the other compilers say.  I added a comment.